src/main/java/com/google/api/tools/framework/aspects/versioning/model/VersionAttribute.java

```diff
@@ -16,14 +16,14 @@
 
 package com.google.api.tools.framework.aspects.versioning.model;
 
+import com.google.api.tools.framework.model.Interface;
```

Comment:
Generally yes. Something like 'organize imports / remove unused' will (for most sane IDEs) not remove the import. You can use the fully qualified name too, its just long and looks ugly.